server/client_manager.py

```diff
@@ -58,6 +58,8 @@ def __init__(self, server, transport, user_id, ipid):
             self.voting_at = 0
             self.is_checked = False
             self.websocket = None
+            self.active_player = False
+            self.new_player = True
```

Comment:
why is this needed?

Reply:
So that we don't log the "a client has actually joined the server" log more than once on a client.